siteimprove/readme.txt

```diff
@@ -28,37 +28,41 @@ Upload the Siteimprove plugin zip file using Wordpress’ ‘Upload Plugin’ fe
 
 = Configuration =
 
-After installing and activating the plugin, you should now see a Siteimprove icon on the right sidebar of the admin page.  
+After installing and activating the plugin, navigate to a piece of content in your WordPress configuration panel, i.e. "Pages". While you are editing it, you should now see a Siteimprove icon on the right sidebar of the admin page.  
 
-Click on ‘Login’ and enter your Siteimprove username and password to access the features.  
+Click on ‘Login’ and enter your Siteimprove username and password to access Siteimprove.  
 
 A new Siteimprove menu option will also appear on the left menu bar. Click on this menu option to continue configuring the plugin once you’ve logged in.  
 
 The token field is automatically filled and should not be changed unless a new token is required. In such cases, please click on “Request new token” to generate a new token. 
 
-API Username (Email) and API Key are API credentials that should be filled in if you have purchased the ‘Prepublish’ add-on feature. When provided correctly, you should be able to begin utilizing Prepublish.  
+Public URL can be optionally filled in, if for any reason, your published pages are not on the same URL as your configuration panel. 
+
+When you download the plugin, you will be automatically be using the latest experience. If for some reason, you would like to use previous look of the Siteimprove  plugin, uncheck "Use latest experience" in the configuration panel. 
+
+API Username (Email) and API Key are API credentials that should be filled in if you have purchased the ‘Prepublish’ add-on feature. When provided correctly, you should be able to begin utilizing Prepublish when you are previewing a page. 
 
 Click on ‘Save’ once you have filled out the relevant fields.  
 
 
 = Usage =
 
-Once you have completed configuring the Siteimprove plugin, clicking on the Siteimprove icon on the right sidebar will open the Siteimprove plugin, which will provide you with a full overview of your global site status and your current Digital Certainty Index (DCI) score.  
+Once you have completed configuring the Siteimprove plugin, if you are at a piece of content in the configuration panel, clicking on the Siteimprove icon on the right sidebar will open the Siteimprove plugin, which will provide you with the content's current issue and the page's current Digital Certainty Index (DCI) score.  
```

Comment:
After review from Product Marketing:
> Once you have completed configuring the Siteimprove plugin and you are on a content page in the configuration panel, clicking on the Siteimprove icon in the right
sidebar will open the Siteimprove plugin, which will provide you with the content's current issues and the page's current Digital Certainty Index (DCI) score.